pages/common/birdc.md

```diff
@@ -0,0 +1,37 @@
+# birdc
+
+> Bird remote control.
+> Command tool to retrieve information like routes from bird and perform configurations during runtime.
+> More information: <https://bird.network.cz/>.
```

Comment:
NIT:
```suggestion
> More information: <https://bird.network.cz>.
```

Reply:
Is there a particular reason for this? In other pages (for example `tshark`) this is not the case either.

Reply:
We try to maintain a standard. Note that this is pointed as a nitpick and not something as a blocker.
The idea is to follow something, e.g: - do we have a trailing `/` or not.
As per my opinion, the consensus has been to drop the trailing `/`.